### DISCUSSION TITLE
how can I use "tts_models/multilingual/multi-dataset/xtts_v2" without voice cloning?

### DISCUSSION THREAD

**Discussion by trappedinspacetime** [coqui-ai/TTS#3619] [General Q&A]
I use 

         tts --model_name tts_models/multilingual/multi-dataset/xtts_v2 --text "Bugün okula gitmek istemiyorum." --speaker_wav untitled.mp3 --language_idx tr
         
by using a sample speaker voice.
 I don't want to use voice cloning. Is it possible?

**Comment by akabakov** [comment]
> I use
> 
> ```
>      tts --model_name tts_models/multilingual/multi-dataset/xtts_v2 --text "Bugün okula gitmek istemiyorum." --speaker_wav untitled.mp3 --language_idx tr
> ```
> 
> by using a sample speaker voice. I don't want to use voice cloning. Is it possible?

You can try the next approach:
`tts --text "Text" --out_path ./out.wav --model_name tts_models/multilingual/multi-dataset/xtts_v2 --speaker_idx "Ana Florence" --language_idx en`

**Reply by trappedinspacetime** [reply]
@akabakov thank you for responding. When I run it, I get:

       ValueError:  [!] Missing speakers.json file path for selecting speaker Ana Florence.Define path for speaker.json if it is a multi-speaker model or remove defined speaker idx.
